lab/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="test2.js"> </script>
+</head>
+<body>
+    <button onclick="add(10,5)"> click me </button>
+</body>
+</html>
```

Comment:
we should add an empty space at the end of all files in Github

Reply:
i didnt know about the convention